benefit/data/sheets/CaseTypeTab/CaseTypeTab.json

```diff
@@ -266,7 +266,7 @@
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "fta_po_details", "TabLabel": "FTA PO Details", "TabDisplayOrder": 24, "CaseFieldID": "presentingOfficersDetails", "TabFieldDisplayOrder": 1},
   {"LiveFrom": "", "CaseTypeID": "Benefit", "AccessProfile": "caseworker-sscs", "TabID": "fta_po_details", "TabLabel": "FTA PO Details", "TabDisplayOrder": 20, "CaseFieldID": "presentingOfficersHearingLink", "TabFieldDisplayOrder": 2},
 
-  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "listing_requirements_judge", "TabLabel": "Listing Requirements", "TabDisplayOrder": 24, "CaseFieldID": "isConfidentialCaseLabel", "TabFieldDisplayOrder": 1, "TabShowCondition": "defaultListingValues=\"*\" OR regionalProcessingCenter.hearingRoute=\"gaps\"",  "FieldShowCondition": "isConfidentialCase =\"Yes\""},
+  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "listing_requirements_judge", "TabLabel": "Listing Requirements", "TabDisplayOrder": 24, "CaseFieldID": "isConfidentialCaseLabel", "TabFieldDisplayOrder": 1, "TabShowCondition": "regionalProcessingCenter.hearingRoute=\"listAssist\" OR regionalProcessingCenter.hearingRoute=\"gaps\"", "FieldShowCondition": "isConfidentialCase =\"Yes\""},
```

Comment:
there are lot of cases in prod where hearing route is not set. Please check prod cases to see if all of them have this `regionalProcessingCenter.hearingRoute` field set

Reply:
As far as Ive been told by POs, the route should always have one of those 2 values by the point they wish to see the tab
Is that contrary to your understanding?

Reply:
just worth confirming that in DB we don't have any cases which doesn't has that value. Once confirmed we are good to go. If they are not set for dormant cases you can confirm with PO and handle it accordingly.